### PR TITLE
Create loopback-sdk-angular-cli.travis.yml

### PR DESCRIPTION
**Pull request by genisysram**

### CODE REVIEW

File: loopback-sdk-angular-cli.travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : loopback-sdk-angular-cli
+# Source Repo         :https://github.com/strongloop/loopback-sdk-angular-cli.git
```
Comment (gururajrkatti) [inline loopback-sdk-angular-cli.travis.yml:4]: Pl. correct the indentation

File: loopback-sdk-angular-cli.travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# ----------------------------------------------------------------------------
+
+
+sudo: false
+arch:
+    - amd64
+    - ppc64le
+language: node_js
+node_js:
+  - "8"
```
Comment (gururajrkatti) [inline loopback-sdk-angular-cli.travis.yml:20]: Add the comment mentioning the reason for node 6

File: loopback-sdk-angular-cli.travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# ----------------------------------------------------------------------------
```
Comment (gururajrkatti) [inline loopback-sdk-angular-cli.travis.yml:1]: Correct the file location

File: loopback-sdk-angular-cli.travis.yml (hunk 2 above)
Comment (genisysram) [reply loopback-sdk-angular-cli.travis.yml:20]: @gururajrkatti 
nodejs is failing in master repo(amd 64) , and i didn't get any handy  solution to fix it , so i have removed it from the yml file.(only kept the latest version)
pls check  the  https://travis-ci.com/github/genisysram/loopback-sdk-angular-cli/jobs/465805819 .

**Comment by genisysram** [comment]
@gururajrkatti 
updated the changes

**Review by gururajrkatti — APPROVED** [review]
LGTM

**Pull request merged**